concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -240,7 +240,7 @@ def test_requestor_should_receive_subtask_results_from_concent_with_correct_keys
         with freeze_time("2018-02-05 10:00:29"):
             response = self.client.post(
                 reverse('core:receive'),
-                data                            = '',
+                data                            = self._create_client_auth_message(self.DIFFERENT_REQUESTOR_PRIVATE_KEY, self.DIFFERENT_REQUESTOR_PUBLIC_KEY),
```

Comment:
One more thing that about the tests. How about you write 4 wrapping functions for `_create_client_auth_message`, like this:
```
def _create_client_auth_message_provider():
    return self._create_client_auth_message(self.DIFFERENT_REQUESTOR_PRIVATE_KEY, self.DIFFERENT_REQUESTOR_PUBLIC_KEY)
```

etc. It shouldn't take much to write them, and they can be replaced in all filed with single grep.

Reply:
Done

Reply:
Missing in this file.

Reply:
Done